### DISCUSSION TITLE
Searching for people to add to a repo should not be case sensitive

### DISCUSSION THREAD

**Discussion by jcollum-nutrien** [community/community#36577] [Repositories]
![image](https://user-images.githubusercontent.com/99687439/196506424-575924dd-fc25-4718-ae3c-065a8e2d26a1.png)

![image](https://user-images.githubusercontent.com/99687439/196506455-3f181555-384e-4e14-ad7c-e06b14c2b0d3.png)

I shouldn't have to use a capital B to find someone in my org named Brandon -- if I use brandon or Brandon it should find someone named Brandon. I can see the use case of putting exact case matches higher in the results.

**Comment by ossanna16** [comment]
Hey there @jcollum-nutrien and thank you for the feedback! I'm glad you found our community 🙂
